src/Tomcat/TomcatController.ts

```diff
@@ -51,7 +53,7 @@ export class TomcatController {
                     this.startServer(server);
                 }
             }
-            Utility.browse(true, httpPort, warPackage.label);
+            opn(`${Constants.LOCALHOST}:${httpPort}/${warPackage.label}`);
```

Comment:
To handle any url, I suggest using a library. We used to use URL.js but please check whether it's already part of node.js now.

Reply:
URL.js Added in: v7.0.0.

Reply:
Update as ```opn(new URL(warPackage.label, `${Constants.LOCALHOST}:${httpPort}`).toString());```

But still need to construct the port.